deeplake/core/dataset/deeplake_query_tensor.py

```diff
@@ -106,6 +106,10 @@ def htype(self):
             htype = f"link[{htype}]"
         return htype
 
+    @htype.setter
```

Comment:
Did we add a test for this?

Reply:
this is on query dataset side of things, might be an unnecessary test